apps/web/app/api/v1/webhooks/[webhookId]/lib/webhook.test.ts

```
@@ -33,9 +34,14 @@ vi.mock("@/lib/utils/validate", () => ({
   },
 }));
 
+vi.mock("@/lib/cache", () => ({
+  cache: (fn: Function) => fn,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)